helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ClusterEventMonitor.java

```diff
@@ -26,10 +26,10 @@
 
 import com.codahale.metrics.Histogram;
 import com.codahale.metrics.SlidingTimeWindowArrayReservoir;
-import org.apache.helix.monitoring.mbeans.dynamicMBeans.DynamicMBeanProvider;
-import org.apache.helix.monitoring.mbeans.dynamicMBeans.DynamicMetric;
-import org.apache.helix.monitoring.mbeans.dynamicMBeans.HistogramDynamicMetric;
-import org.apache.helix.monitoring.mbeans.dynamicMBeans.SimpleDynamicMetric;
+import org.apache.helix.monitoring.common.mbeans.dynamicMBeans.DynamicMBeanProvider;
+import org.apache.helix.monitoring.common.mbeans.dynamicMBeans.DynamicMetric;
+import org.apache.helix.monitoring.common.mbeans.dynamicMBeans.HistogramDynamicMetric;
+import org.apache.helix.monitoring.common.mbeans.dynamicMBeans.SimpleDynamicMetric;
```

Comment:
The original package naming was intentional in order to not cause existing applications to have to change import statements (code change). Why is this needed? Are users importing multiple modules at the same time?

Reply:
When users import helix-core, since helix-core depends on helix-common, helix-common.jar and its transitive dependencies are also imported.
I understand the purpose of not changing packaging naming, but packaging dependencies into helix-core is not well handled, which causes the duplicate class issue.